### ISSUE TITLE
pushing to LOAD_PATH does not run before `using Foo`

### ISSUE THREAD

**Issue by jakobnissen** [fonsp/Pluto.jl#323]
To reproduce, do the following:
* Have a Julia package `Foo.jl` in some directory `/foo/bar`, not in the default `LOAD_PATH`
* Create a new notebook
* In the first cell, type `push!(LOAD_PATH, "/foo/bar")`
* In the second cell, type `using Foo`
* Restart the notebook
* Observe
```
ArgumentError: Package Foo not found in current path:

- Run `import Pkg; Pkg.add("Foo")` to install the Foo package.
```

**Comment by fonsp** [comment]
Is `Pkg.develop` a workaround?

**Comment by fonsp** [comment]
Cells that use `using`, `import` or `include` are always prioritized over other cells when computing the topological order, so you need to do

```julia
begin
    push!(LOAD_PATH, "...")
    using Foo
end
```

**Comment by fonsp** [comment]
Same with `Pkg.add` and `Pkg.develop`:
```julia
begin
    Pkg.add("Foo")
    using Foo
end
```

**Comment by jakobnissen** [comment]
I see the rationale for putting the `include` cells at the top. But perhaps, a special case could be made for LOAD_PATH? For example, if LOAD_PATH was detected in a cell, a warning could be printed that it should be used with `begin end`, like in your proposed fix?

In the current state, it's not very clear why it errors.

**Comment by fonsp** [comment]
Can you check if it is fixed in `0.11.10`?

**Comment by jakobnissen** [comment]
It works for me on version 0.11.10. Thanks for the fix!
